test/selenium/signInPage.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var SignedInPageView = require('./lib/view/signed_in/view.js');
+var signedInPageView;
+
+function SignInPage(driver) {
```

Comment:
Nit: The naming is a bit redundant with `SignedInPageView`. How about something like `SetupWebapp`?